src/components/organisms/Video/VideoParticipant.scss

```diff
@@ -0,0 +1,52 @@
+.video-participant {
```

Comment:
Yeah, I cherry-picked the commits from your PR

Reply:
Leaving this one unresolved. Will resolve the similar ones

Reply:
Makes sense.

Can you please update the description of this PR to note that it's dependent on #1155 being merged first since you're using code from it. Otherwise it negates the point of #1155 existing in the first place.

I believe the method I suggested when you asked about it was that we could merge #1155 as is, then you could use it from `staging` in this PR, and we could handle 'wiring it up' to the rest of the codebase in a followup PR.. but seemingly that wasn't followed?

Reply:
https://github.com/sparkletown/sparkle/pull/1155 was merged into staging

Reply:
With additional bug fixes made/merged in https://github.com/sparkletown/sparkle/pull/1354